### DISCUSSION TITLE
Enterprise associated dev account randomly suspended by Github!

### DISCUSSION THREAD

**Discussion by arvindrag** [community/community#116561] [Enterprise] [answered]
My enterprise related account which is associated with my place of work was suddenly and suspended without any cause or explanation!
Attempting to log in merely shows the Account Suspended warning. To do this without warning/explanation/cause for an enterprise product seems bizarre! No warning or cause is available in the org's audit log.
<img width="705" alt="image" src="https://github.com/community/community/assets/7065454/8b19bd92-ba37-4b90-963c-6c12a50d020c">

**Comment by gpereiradev** [comment]
try logging in again, it worked

**Reply by ItsAbhi2004** [reply]
Hello, my GitHub account has been suspended for an alleged Terms of Service violation. However, I did not receive any email explaining the reason. Could you please tell me why my account was suspended and whether it can be restored?

My GitHub username - raj933955

Email- rajkrshawjee@gmail.com

**Comment by arvindrag** [comment]
As others might have noticed - this was silently unsuspended. No email - or explaination

**Comment by dannykansas** [comment]
Keeping my support ticket open despite being unsuspended. This broke just about everything for a few hours; would like to know why / how to avoid in the future.

**Reply by arvindrag** [reply]
Same. We just noticed that our deploy-bot was suspended and silently unsuspended as well

**Comment by github-staff** [comment, marked as answer]
Thank you for your patience. Our Platform Health team has investigated this issue and found the root cause to be an error in how malicious behavior was flagged, causing the suspension of several accounts. This error has been resolved and the affected accounts have been reinstated.

We apologize for the inconvenience this caused. If your account is still currently suspended, please reach out to Support for further investigation.

**Reply by arvindrag** [reply]
Fair enough - closing this out - but I'd appreciate a more comprehensive answer in the actual support ticket (including how to prevent suspension-without-cause in the future)- especially since our deploy-bot was found to be suspended as well

**Comment by devzero25** [comment]
Hello have same issue for account Hussseinkizz and really think there might have been a mistake as well as nothing changed really in usual account usuage!

**Reply by MartinPirate** [reply]
did you get any help?

**Comment by whisdev** [comment]
I have two github accounts for personal reason. One is this and the other is businees account named https://github.com/taprwhiz
This one connected to my workplace was suddenly suspended without any warning or explanation.
When I try to log in, it simply shows “Account Suspended,” and there’s no additional information in the organization’s audit log. Since this account is tied to our work, we’re concerned about the sudden restriction and would appreciate any clarification or next steps to resolve it.

**Comment by Kakashi-17** [comment]
Hello GitHub Support Team,

My main GitHub account (DevGopi-17) has been suspended due to an alleged Terms of Service violation. I was unexpectedly logged out and currently cannot access the account. Additionally, I am not receiving any related emails regarding this suspension.

I am submitting this request from an alternate account because I do not have access to the suspended account. I am not aware of any violation and kindly request a review and clarification of the reason for this action.

If any issue occurred unintentionally, I am fully willing to resolve it and comply with all GitHub policies. I respectfully request reinstatement of my account.

Primary email associated with the account: gopiprajapati2005@gmail.com

Username: DevGopi-17

Thank you for your assistance.
<img width="490" height="514" alt="Screenshot 2026-03-02 at 10 32 38 AM" src="https://github.com/user-attachments/assets/77e43969-9048-4781-b117-2001c8f1173f" />
<img width="647" height="468" alt="Screenshot 2026-03-02 at 10 34 01 AM" src="https://github.com/user-attachments/assets/0c13a100-6cad-4ce1-8769-a9ebad924680" />
